src/Components/CreateImageWizard/CreateImageWizard.js

```diff
@@ -151,6 +152,13 @@ const onSave = (values) => {
   }
 
   if (values['target-environment']?.azure) {
+    const upload_options =
+      values['azure-type'] === 'azure-type-source'
+        ? { source_id: [values['azure-sources-select']] }
```

Comment:
Azure only supports sharing to a single source, so let's just use `source_id: values['azure-sources-select']` instead of putting it in a list.

Reply:
ah, yeah, makes sense, no way to target multiple accounts for azure, right? :)

Reply:
Fixed :)